GetIntoTeachingApiTests/Models/TeachingEventsByTypeTests.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace GetIntoTeachingApiTests.Models
+{
+    public class TeachingEventsByTypeTests
+    {
```

Comment:
Should this spec be empty?

Reply:
Yeah the model only has getters/setters that are tested indirectly, no real value testing them explicitly but I like to have the test file there for if/when its needed.